go/test/endtoend/vtgate/schematracker/unsharded/st_unsharded_test.go

```diff
@@ -176,14 +168,63 @@ func TestNewUnshardedTable(t *testing.T) {
 	utils.Exec(t, conn, `drop table new_table_tracked`)
 
 	// waiting for the vttablet's schema_reload interval to kick in
-	expected = `[[VARCHAR("dual")] [VARCHAR("main")]]`
-	if vtgateVersion >= 17 {
-		expected = `[[VARCHAR("main")]]`
-	}
+	expected = `[[VARCHAR("main")]]`
 	utils.AssertMatchesWithTimeout(t, conn,
 		"SHOW VSCHEMA TABLES",
 		expected,
 		100*time.Millisecond,
 		30*time.Second,
 		"new_table_tracked not in vschema tables")
 }
+
+// TestCaseSensitiveSchemaTracking tests that schema tracking is case-sensitive.
+// This test only works on Linux (and not on Windows and Mac) since it has a case-sensitive file system, so it allows
+// creating two tables having the same name differing only in casing, but other operating systems don't.
+// More information at https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html#:~:text=Table%20names%20are%20stored%20in,lowercase%20on%20storage%20and%20lookup.
+func TestCaseSensitiveSchemaTracking(t *testing.T) {
+	utils.SkipIfBinaryIsBelowVersion(t, 19, "vttablet")
```

Comment:
since we are backporting this fix to all release branch we might get rid of this line in the future

Reply:
We aren't backporting this anymore